.env.example

```diff
@@ -12,3 +12,7 @@ DB_PATH=:memory: #required
 DEPLOY_START=0
 DEPLOY_END=19
 # FORK=mainnet
+TIDY_TEST=false
+TEST_ONLY=false
+NFT_ORACLE_PRICE_EXPIRATION=1800
```

Comment:
nft config doesn't change so much, so if we really need to have config per network. We can put inside market-config